aiida/cmdline/utils/common.py

```diff
@@ -511,16 +512,16 @@ def check_worker_load(active_slots):
     try:
         active_workers = get_num_workers()
     except CircusCallError:
-        echo.echo_critical('Could not contact Circus to get the number of active workers')
+        echo.echo_critical('Could not contact Circus to get the number of active workers.')
```

Comment:
Is there a way to test this path? The `get_num_workers` is another function here so there is nothing getting passed to the method being tested that could be replaced by a raising mock object... what can be done in this cases?

Reply:
Nevemind, I just saw the `test_check_worker_load` [test](https://github.com/aiidateam/aiida-core/pull/5148/files#diff-84765cd313b2c0415db00d64e15b16f12347b2b980d3d71e76829c684bc9a164R69-R86). That monkeypatch thing is black magic. Anyways, can you add also one for this? Otherwise everything else looks good.

Reply:
Done